contestant_containers/arch/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/archlinux
```

Comment:
FROM blackarchlinux/blackarch

Reply:
I updated this to `FROM docker.io/blackarchlinux/blackarch:latest`, which alleviates the forced user selection shown below.

```bash
$ podman build . -t wctf-arch
STEP 1: FROM blackarchlinux/blackarch
? Please select an image: 
  ▸ docker.io/blackarchlinux/blackarch:latest
    quay.io/blackarchlinux/blackarch:latest
```

Reply:
wfm, docker (obviously) defaults to dockerhub, having it specified is fine.